src/docs/markdown/caddyfile/patterns.md

```diff
@@ -194,6 +195,8 @@ If your SPA is coupled with an API or other server-side-only endpoints, you will
 
 ```caddy
 example.com {
+	encode gzip
```

Comment:
Maybe a comment explaining that this is optional? I really, really try to keep examples the bare minimum so that readers know what is relevant, since that's the point of the example.

Reply:
Well it is relevant. There's never a situation where you wouldn't want this on, when you're serving an SPA. It would be terrible if they didn't have compression on, cause then all their JS/CSS/HTML bandwidth would suffer.